test/column/editor.test.tsx

```diff
@@ -21,7 +21,7 @@ describe('Editor', () => {
     userEvent.type(document.activeElement!, '2');
     userEvent.tab();
     expect(screen.queryByLabelText('col1-editor')).not.toBeInTheDocument();
-    expect(getCellsAtRowIndex(0)[0]).toHaveTextContent('2');
+    expect(getCellsAtRowIndex(0)[0]).toHaveTextContent(/^12$/);
```

Comment:
I've replaced all `toHaveTextContent('string')` with a strict regex instead, as otherwise `toHaveTextContent('1')` would be valid for `'1234'`.